framework/files/etc/systemd/system/elemental-system-agent.service.d/override.conf

```diff
@@ -0,0 +1,6 @@
+[Unit]
+After=
+After=cos-setup-network.service
+
+[Service]
+ExecStartPre=-/usr/bin/mkdir -p /usr/local/bin
```

Comment:
Missing newline character

Reply:
This is covered by https://github.com/rancher/elemental/pull/172